demo/views/track-verified-token.hbs

```diff
@@ -0,0 +1,94 @@
+<div class="content flex-grow-1 p-4">
+  <h1 class="mb-4">Track verified token</h1>
+  <div id="error-alert" class="alert alert-danger fade show" role="alert" style="display: none;">
+    <strong>Error:</strong> <span id="error-text">Something went wrong. Check the developer console for details.</span>
+  </div>
+
+  <p>Use the Radar Verify app to verify the user's current location. Returns a JSON Web Token (JWT). Verify the JWT server-side using your secret key.</p>
+
+  <div class="mt-5 mb-3 form-row">
+    <label for="user-id" class="col-form-label" style="width: 128px;">User ID</label>
+    <div>
+      <input type="text" class="form-control" id="user-id" value="test-user-id" style="width: 250px;">
+    </div>
+  </div>
+
+  <div>
+    <button id="track-verified" type="button" class="btn btn-primary" disabled>Call track verified token</button>
+  </div>
+
+  <div id="loader" class="spinner-border text-primary mt-4" role="status" style="display: none;">
+  </div>
+
+  <div id="response-details" style="display: none;">
+    <h6 class="mt-4">Token</h6>
+    <div class="mt-2">
+      <pre style="white-space: pre-wrap"><code id="token" class="code-sample language-json"></code><pre>
+    </div>
+  </div>
+
+  <div class="mt-4">
```

Comment:
We can probably omit the code samples on these. These should probably live in the docs (I only added them as a temporary solution until we go the UI kits in the docs).

Reply:
I've actually found it helpful to be able to show the code sample when using this page in demos (which I'm doing a lot), but we can always remove later